.github/auto-assignees.yml

```diff
@@ -0,0 +1,28 @@
+---
+# This assigns a PR to its author
+addAssignees: author
+
+reviewers:
+  # The default reviewers
+  defaults:
+    - maintainers
+
+  groups:
+    maintainers:
+      - carlisia
+      - jenting
+
+    other-maintainers:
```

Comment:
If we set the number of reviewers to 2, would anyone from this group be selected to review?

Reply:
You're on the right track. No, it would never overflow into the "other-maintainers" because the default would satisfy the condition.

Reply:
Actually, reading the instructions again, I *think* if we set to 2 then it will randomly pick from either group. It would be ideal if it could pick 1 from each group so that way we could always have at least 1 of the more regular helm maintainers in the rotation. We can try this and see if it works this way or not.